radis/levels/partfunc.py

```diff
@@ -1170,6 +1170,8 @@ def __init__(self,species):
 
     def _at(self, T):
```

Comment:
if not done already, add a test for `PartFuncKurucz` in `radis/test/levels/test_partfunc.py` 

(making sure it works; and assert that you find the expected partition functions for a few atoms of your choice. You'll ensure it never fails)

Reply:
I have already added such a test in radis/test/io/test_partfunc.py , I will move it to radis/test/levels/test_partfunc.py